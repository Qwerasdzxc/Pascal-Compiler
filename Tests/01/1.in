171.925 693.365

