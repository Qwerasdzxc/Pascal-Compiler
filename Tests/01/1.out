865.29 -521.44 0.25